besu/src/main/java/org/hyperledger/besu/controller/BesuController.java

```diff
@@ -241,6 +247,9 @@ private BesuControllerBuilder createConsensusScheduleBesuControllerBuilder(
       if (configOptions.isIbft2()) {
         originalControllerBuilder = new IbftBesuControllerBuilder();
       } else if (configOptions.isIbftLegacy()) {
+        // TODO remove this warning once ibf1 has been deprecated
+        LOG.warn(
+            "IBFT1 is being deprecated and will be removed in a future release. Consider using QBFT instead of IBFT1 or using GoQuorum instead of Besu if you need to use IBFT1");
```

Comment:
Putting this in the IbftLegacyBesuControllerBuilder constructor would be a better place. Would also remove the duplication.